webapp/src/js/bootstrapper/index.js

```diff
@@ -224,53 +224,70 @@
       .catch(() => true);
 
     let isInitialReplicationNeeded;
-    Promise.all([swRegistration, testReplicationNeeded(), setReplicationId(POUCHDB_OPTIONS, localDb)])
-      .then(function(resolved) {
+    Promise
+      .all([swRegistration, testReplicationNeeded(), setReplicationId(POUCHDB_OPTIONS, localDb)])
+      .then(resolved => {
         purger.setOptions(POUCHDB_OPTIONS);
         isInitialReplicationNeeded = !!resolved[1];
 
         if (isInitialReplicationNeeded) {
+          window.startupTimes.replicationStarted = performance.now();
```

Comment:
As above, can we store this as a local variable and only store the duration in the `window.startupTimes` object?